channels/chan_simpleusb.c

```diff
@@ -3706,6 +3724,7 @@ static struct chan_simpleusb_pvt *store_config(const struct ast_config *cfg, con
 		CV_BOOL("preemphasis", o->preemphasis);
 		CV_UINT("duplex3", o->duplex3);
 		CV_UINT("clipledgpio", o->clipledgpio);
+		CV_BOOL("legacyaudioscaling", o->legacyaudioscaling);
```

Comment:
What do you think about only updating the "legacyaudioscaling" variable in the .conf file when enabled?  Asking because I'm thinking about the future when the default value would be not enabled and when there's no need to call attention to the value.

Reply:
Although the name of this function is store_config(), from what I can tell all it does is load parameters at startup, ie. does not appear to store anything to any file. If you are seeing something other than that happening there let me know.

Reply:
Yes, my bad ... and a poor function name.  The function that updates the .conf file is tune_write().